src/wallet/bdk_wallet/wallet.rs

```diff
@@ -25,3 +36,191 @@ pub struct Wallet {
     rpc: Client,
     wallet_file_path: PathBuf,
 }
+
+impl Wallet {
+    pub fn init(
+        wallet_file_path: PathBuf,
```

Comment:
Instead of Wallet file path, a better approach is to take in a `data_dir` path. As there will be two files, bdk_store and meta_store, each of them should be added inside the `data_dir`, like `meta_store path = data_dir/meta_store`, `bdj_store path = data_dir/bdk_store`. The file names should be constant and not user-defined. The user will only specify the parent wallet data directory.